src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
```

Comment:
if it's a util class, it's better to make the default constructor private, so no one could create an instance of it